### PR TITLE
Update plank dashboard to reference prow-controller-manager

### PR DESCRIPTION
**Pull request by alvaroaleman**

### CODE REVIEW

File: config/prow/cluster/monitoring/mixins/grafana_dashboards/plank.jsonnet [hunk 1]
```diff
@@ -23,7 +23,7 @@ dashboard.new(
   template.new(
     'cluster',
     'prometheus',
-    'label_values(prowjobs{job="plank"}, cluster)',
+    'label_values(prowjobs{job="prow-controller-manager"}, cluster)',
```
Comment (spiffxp) [inline config/prow/cluster/monitoring/mixins/grafana_dashboards/plank.jsonnet:26]: So we don't lose history
```suggestion
    'label_values(prowjobs{job=~"plank|prow-controller-manager"}, cluster)',
```

**Review by fejta — APPROVED** [review]

**Review by spiffxp — COMMENTED** [review]
thanks for this, can we keep plank included so we don't lose history?

File: config/prow/cluster/monitoring/mixins/grafana_dashboards/plank.jsonnet (hunk 1 above)
Comment (spiffxp) [reply config/prow/cluster/monitoring/mixins/grafana_dashboards/plank.jsonnet:26]: (this is in multiple places I just felt like it'd be easier for you to search-replace than me to make multiple suggetions)

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm
Thanks!

**Pull request merged**